src/components/organisms/AdminChallengeOpen/index.tsx

```diff
@@ -84,10 +82,14 @@ function AdminChallengeOpen(): React.ReactElement {
         </PhotoUpload>
       </div>
       <div className="check__container">
-        <CheckBox id="checkBox" className="check__checkbox" checked={check} onChange={checkHandler} />
+        <CheckBox id="checkBox" className="check__checkbox" checked={isChecked} onChange={checkHandler} />
         <label htmlFor="checkBox" className="check__exp">
-          {check ? <img src={checkOn} className="check__icon" /> : <img src={checkOff} className="check__icon" />}위
-          내용을 확인하셨습니까?
+          {isChecked ? (
+            <img src={CheckOnIcon} className="check__icon" />
+          ) : (
+            <img src={CheckOffIcon} className="check__icon" />
+          )}
```

Comment:
```suggestion
        <img src={isChecked ? CheckOnIcon : CheckOffIcon} className="check__icon" />
```
태그나 클래스나 안의 내용이 바뀌지 않고 아이콘만 바뀌는거라면 이렇게 처리하는게 더 깔끔할것같네요! 다른 코드에도 이런 코드가 꽤 있던 것 같아요